tests/_internal/clients/test_pds.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from py42._internal.clients.pds import PreservationDataServiceClient
+
+
+class TestPreservationDataServiceClient(object):
+    @pytest.fixture
+    def mock_session(self, mock_session, successful_response):
```

Comment:
might be worth putting this as a `mock_successful_session` in a conftest

Reply:
Moved.